### PR TITLE
Swift: tracer integration

### PR DESCRIPTION
**Pull request by AlexDenisov**
This was tested locally. Proper integration will come with an internal PR.

### CODE REVIEW

File: swift/extractor/main.cpp [hunk 1]
```diff
@@ -16,6 +16,14 @@ class Observer : public swift::FrontendObserver {
  public:
   explicit Observer(const codeql::SwiftExtractorConfiguration& config) : config{config} {}
 
+  void parsedArgs(swift::CompilerInvocation& invocation) override {
+    // Original compiler and the extractor-compiler get into conflicts when
+    // both produce the same output files.
+    // TODO: change the final arifact destinations instead of disabling
```
Comment (igfoo) [inline swift/extractor/main.cpp:22]: typo: arifact

Comment (AlexDenisov) [inline swift/extractor/main.cpp:22]:
```suggestion
    // TODO: change the final artifact destinations instead of disabling
```

**Review by redsun82 — APPROVED** [review]
:rocket:

**Review by cklin — APPROVED** [review]
I looked at only the Lua tracing config, and that part is 👍

**Pull request merged**